kk/models/hearing.py

```diff
@@ -45,3 +47,10 @@ class HearingImage(BaseImage):
 
 class HearingComment(BaseComment):
     hearing = models.ForeignKey(Hearing, related_name="comments")
+
+
+def hearing_n_comments_bump(sender, instance, using, **kwargs):
+    instance.hearing.n_comments += 1
```

Comment:
Please use `instance.hearing.recache_n_comments()`, available since https://github.com/City-of-Helsinki/kerrokantasi/commit/014e9ad21bde3980794fb04d15d7a30223c7b1b7